### PR TITLE
PPMMOPL-171/Home Screen

### PR DESCRIPTION
**Pull request by ETLopes**
Create home screen for the creditas Store.

### CODE REVIEW

File: module.tsx [hunk 1]
```diff
@@ -8,20 +8,32 @@ import {createStackNavigator} from 'react-navigation-stack';
 import CoreService from './src/CoreService';
 import ConfigEnvironment from './src/ConfigEnvironment';
 
+import Home from './src/screens/Home';
+import ExternalView from './src/screens/Webview';
+
 //import Icons from './assets/icons';
```
Comment (erickzanardo) [inline module.tsx:14]: We need to see with the UI to see what will be this mini app icon.

File: package.json [hunk 2]
```diff
@@ -32,22 +34,23 @@
     "eslint": "6.5.1",
     "jest": "24.9.0",
     "metro-react-native-babel-preset": "0.58.0",
-    "react-test-renderer": "16.11.0",
+    "mobile-core-api": "git+ssh://git@github.com:Creditas/mobile-core-api.git#0.0.22",
+    "mobile-core-api-mock": "git+ssh://git@github.com:Creditas/mobile-core-api-mock.git#0.0.18",
```
Comment (erickzanardo) [inline package.json:38]: I have just released a new version of this and the mock as well, maybe we should already update here, 0.0.23 and 0.0.19

File: src/screens/Home/index.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  SmallCardArea,
+} from './style';
+
+import {apple, education, sittingGirl} from '../../assets'
+
+const Home = ({navigation}) => {
+  const routes = {
+    more: 'https://www.creditas.com/store',
+    apple: 'https://www.creditas.com/store/apple',
+    education: 'https://creditas.wl.descubraomundo.com',
```
Comment (erickzanardo) [inline src/screens/Home/index.tsx:28]: Should we add a dotenv plugin to store this king of thing? What do you think?

File: src/screens/Home/style.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components/native';
+
+export const Container = styled.View`
+  margin-top: 40px;
+  margin-left: 30px;
+  margin-right: 30px;
```
Comment (erickzanardo) [inline src/screens/Home/style.tsx:6]: Can we use margin-horizontal here, I am not that familiar with styled components

File: package.json (hunk 2 above)
Comment (ETLopes) [reply package.json:38]: Done!

File: src/screens/Home/style.tsx (hunk 4 above)
Comment (ETLopes) [reply src/screens/Home/style.tsx:6]: Done

File: module.tsx (hunk 1 above)
Comment (ETLopes) [reply module.tsx]: Icon added!

**Pull request merged**